src/jsonToCsv.ts

```diff
@@ -0,0 +1,67 @@
+import { createReadStream, createWriteStream } from "fs";
+import { stringifyJsonChunk, prepareRow } from "./utils";
+import { Header, Paths, RowObject } from "./types";
+import JSONStream from "jsonstream";
+import { map } from "event-stream";
+import { Transform } from "stream";
+
+const encoding: BufferEncoding = "utf-8";
+
+const mapPipedData = map((data: RowObject, callback: Function) => {
+  const strigified = stringifyJsonChunk(data);
+  callback(null, strigified);
+});
+
+export const convertFilesWithJson = (header: Header, paths: Paths) => {
+  return new Promise((resolve, reject) => {
+    const pathSplitted = paths.in.split("/");
```

Comment:
Use build in node.js modules to extract the file name.

Reply:
It looks a bit different now. File name comes as argument